composer.json

```diff
@@ -48,12 +48,14 @@
         "silverstripe/recipe-testing": "^2",
         "mikey179/vfsstream": "^1.6"
     },
-    "replace": {
-        "sminnee/phpunit": "*",
-        "sminnee/phpunit-mock-objects": "*"
-    },
     "extra": {
```

Comment:
I think that bit was just there temporarily to allow us to install module that also required Sam's fork

Reply:
Yup, though it does no harm keeping it in, though may cause problems if we take it out.  Would much prefer to just keep it in.

Reply:
What potential problems are you thinking about? I would rather keep things clean and avoid carrying dead weight logic.

Reply:
Any required modules that still have sminnee/phpunit in composer.json

Which should be none.  Probably good having this removed at this stage since if it's a big negative if it's still being isntalled